blazor/file-manager/context-menu.md

```diff
@@ -0,0 +1,322 @@
+    <FileManagerAjaxSettings Url="https://ej2-aspcore-service.azurewebsites.net/api/FileManager/FileOperations"
+                             UploadUrl="https://ej2-aspcore-service.azurewebsites.net/api/FileManager/Upload"
+                             DownloadUrl="https://ej2-aspcore-service.azurewebsites.net/api/FileManager/Download"
+                             GetImageUrl="https://ej2-aspcore-service.azurewebsites.net/api/FileManager/GetImage">
+    </FileManagerAjaxSettings>
+    <FileManagerContextMenuSettings File="@FileItems" Folder="@FolderItems"></FileManagerContextMenuSettings>
+</SfFileManager>
+
+@code {
+    SfFileManager<FileManagerDirectoryContent>? FileManager;
+    public string[] FileItems = new string[] { "Delete", "Download", "Rename", "|", "Details" };
+    public string[] FolderItems = new string[] { "Open", "|", "Cut", "Copy", "Paste"};
+}
+
+```
+
+## Enabling or disabling items
+
+In the Blazor FileManager component, you can enable or disable context menu items by setting the [Disabled](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.FileManager.MenuItemModel.html#Syncfusion_Blazor_FileManager_MenuItemModel_Disabled) value of the [MenuOpened](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.FileManager.FileManagerEvents-1.html#Syncfusion_Blazor_FileManager_FileManagerEvents_1_MenuOpened) event arguments to either **true** or **false**.
+
+In the following example, the **Cut** context menu item is disabled for the folders.
+
+```cshtml
+
+@using Syncfusion.Blazor.FileManager
+
+<SfFileManager TValue="FileManagerDirectoryContent">
+    <FileManagerEvents TValue="FileManagerDirectoryContent"></FileManagerEvents>
+    <FileManagerAjaxSettings Url="https://ej2-aspcore-service.azurewebsites.net/api/FileManager/FileOperations"
+                             UploadUrl="https://ej2-aspcore-service.azurewebsites.net/api/FileManager/Upload"
+                             DownloadUrl="https://ej2-aspcore-service.azurewebsites.net/api/FileManager/Download"
+                             GetImageUrl="https://ej2-aspcore-service.azurewebsites.net/api/FileManager/GetImage">
+    </FileManagerAjaxSettings>
+    <FileManagerEvents TValue="FileManagerDirectoryContent" MenuOpened="MenuOpened"></FileManagerEvents>
+</SfFileManager>
+
+@code {
+
+    public void MenuOpened(MenuOpenEventArgs<FileManagerDirectoryContent> args)
+    {
+        for (int i = 0; i < args.FileDetails.Count(); i++)
+        {
+            if (!args.FileDetails[i].IsFile)
+            {
+                for (int j = 0; j < args.Items.Count(); j++)
+                {
+                    if (args.Items[j].Text == "Cut")
+                    {
+                        args.Items[j].Disabled = true;
+                    }
+                }
+            }
+            else
+            {
+                for (int j = 0; j < args.Items.Count(); j++)
+                {
+                    if (args.Items[j].Disabled = true;)

```

Comment:
This will assign a 'true' value. Check proper condition @PrasanthMadhaiyan

Reply:
Done